simulator.py

```diff
@@ -73,8 +59,12 @@ def fetch_dialog_state(self, phone_number: str) -> DialogState:
             )
 
     def get_next_unstarted_drill(self):
+        language = self.fetch_dialog_state(PHONE_NUMBER).user_profile.language
         unstarted_drills = [
-            code for code in DRILLS.keys() if DRILLS[code].slug not in STARTED_DRILLS.values()
+            code
+            for code in DRILLS.keys()
+            if DRILLS[code].slug not in STARTED_DRILLS.values()
+            and DRILLS[code].slug.endswith(language)
```

Comment:
oh this is in the simulator. Is that why?

Reply:
Yeah! We have to somehow distinguish between the english and spanish drills because we no longer localize drills in the dialog engine. this seemed like the simplest way to get the simulator into the new pattern

Reply:
Really there should be a new datastructure in drills.json but this seems fine too